src/compat/posix/include/sys/mman.h

```diff
@@ -28,27 +30,25 @@ __BEGIN_DECLS
 
 #define MAP_FAILED (void*)-1
 
-#if 1
-#include <errno.h>
-static inline void  *mmap(void *addr, size_t len, int prot, int flags, int fd, off_t off) {
-	// ToDo: implement for InitFS files
-	(void)addr;
-	(void)len;
-	(void)prot;
-	(void)flags;
-	(void)fd;
-	(void)off;
-	//printf(">>> mmap(%i)\n",fd);
-	errno = EPERM;
-	return NULL;
-}
+extern struct void *phymem_alloc(size_t);
+extern int mprotect(void *, size_t, int);
 
+// todo: implement for InitFS files
+static inline void *mmap(void *addr, size_t len, int prot, int flags, int fd, off_t off) {
```

Comment:
Please consider moving the implementation to .c file instead of .h. Before it became full-weight function, that little stub was pretty OK to be in header, but now the function deserves better.